app/assets/images/PIV-CAC-round.svg

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We can crunch these files a fair bit by using something like [svgo](https://www.npmjs.com/package/svgo):

```
$ svgo app/assets/images/PIV-CAC-round.svg 

PIV-CAC-round.svg:
Done in 45 ms!
18.907 KiB - 64.2% = 6.773 KiB
```

If you want, I can go ahead and push them up.

Reply:
Now that I have updated the file name, yeah go ahead. Thanks.

Reply:
Done in 327c1a2.